src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -26,7 +25,7 @@ import { getFirstUseCaseOfFeatureConfigs } from '../../utils';
 import { WorkspaceUseCase } from '../../types';
 import { validateWorkspaceColor } from '../../../common/utils';
 import { WorkspacePickerContent } from '../workspace_picker_content/workspace_picker_content';
-
+import './workspace_menu.scss';
```

Comment:
Nit: keep the empty line
```suggestion
import './workspace_menu.scss';

```